uwblueprint/childrens-aid-society
# Implement formik

## Notion ticket link

[Implement Formik](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* wrapped each form with formik within the form files
* onSubmit, we set the different form details using state (which exist in the intake page)



## Steps to test
1. checkout branch
2. check that values are being submitted onSubmit (can add a console.log(values) in each of the onSubmits)



## What should reviewers focus on?
* double check whether or not we want to "submit"/save the information when we click the previous and next buttons


## Checklist
